src/pages/rss.xml.ts

```diff
@@ -0,0 +1,100 @@
+// A fast, cached build-time only RSS.
+// Currently returns title, pubDate, link, guid & description.
+// Description is truncated to 200 chars.
+// Image support exists, just needed to figure out the fuckery of XML.
+
+import rss from "@astrojs/rss";
+import type { APIRoute } from "astro";
+import { getCollection, type CollectionEntry } from "astro:content";
+
+const baseUrl = 'https://nn1.dev';
+
+let cachedXML: string | null = null;
+type SpotlightEntry = CollectionEntry<"spotlight">;
+
+export const GET: APIRoute = async () => {
+  if (cachedXML) {
```

Comment:
We can remove this one and rely on the Netlify CDN caching controlled by headers. We already have Netlify adapter connected to the project. I think adding single header, can do the job for us. I'm not sure about the values, but look like day of caching and week for revalidation is ok starting point. Do you agree?

```js
    return new Response(cachedXML, {
       headers: {
          "Content-Type": "application/rss+xml; charset=utf-8",
          "Netlify-CDN-Cache-Control": "public, durable, s-maxage=86400, stale-while-revalidate=604800"
        },
     });


```